### PR TITLE
Some fixes that make it run in docker

### PR DESCRIPTION
**Pull request by jumpingGrendel**
* Added EMAIL_PORT to env file
* Added unicodecsv to requirements.txt
* Added flask startup to supervisor
* Changed 'imports' to CELERY_IMPORTS in celeryconfig.py

### CODE REVIEW

File: celeryconfig.py [hunk 1]
```diff
@@ -10,4 +10,4 @@
 
 broker_url = getenv('BROKER_URL')
 result_backend = getenv('CELERY_RESULT_BACKEND')
-imports = ("sms_to_channel_id")
+CELERY_IMPORTS = ("sms_to_channel_id")
```
Comment (bugzPDX) [inline celeryconfig.py]: Celery will complain if you mix case. Lower case is apparently the new hotness.

Comment (jumpingGrendel) [reply celeryconfig.py]: it wouldn't start until i changed it :/

**Comment by bugzPDX** [comment]
Can you make sure your version matches what is on the server?

**Comment by jumpingGrendel** [comment]
celery is unpinned in requirements.txt.  Is there a particular version it should be set to?  If so I will set it to that in requirements and that might solve the lowercase/uppercase issue...

**Comment by bugzPDX** [comment]
I think it's 4.3.0. Whatever the latest is. I'm in the car so can't check.

**Comment by jumpingGrendel** [comment]
you were right, i was wr^H^H^H^H

**Comment by jumpingGrendel** [comment]
I had to change all the vars to lowercase

**Pull request merged**